src/lib.rs

```diff
@@ -169,8 +169,8 @@ impl Target {
 
     fn triple(&self) -> &str {
         match *self {
-            Target::BuiltIn { ref triple } => triple,
-            Target::Custom { ref triple } => triple,
+            Target::BuiltIn { ref triple, .. } => triple,
+            Target::Custom { ref triple, .. } => triple,
```

Comment:
future proofing?

Reply:
Uh whoops that was from when it used `libc` previously. This needs to be removed.